<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_table</class>
 <widget class="QDialog" name="add_table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建表</string>
  </property>
  <widget class="QPushButton" name="add_table_ok">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认创建</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_table_clear">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>重置内容</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>新表名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newtable_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>631</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加列</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除列</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>200</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>主键</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
